src/api/java/com/minecolonies/api/util/constant/GuardConstants.java

```diff
@@ -110,7 +110,7 @@
     /**
      * Basic bonus hp for knights
      */
-    public static final int KNIGHT_HP_BONUS = 5;
+    public static final int KNIGHT_HP_BONUS = 15;
```

Comment:
Why is this changed in this PR?

Reply:
didnt want to make a new PR for two lines :D